src/strategies/index.ts

```diff
@@ -20,8 +22,10 @@ export default {
   'eth-balance': ethBalance,
   'maker-ds-chief': makerDsChief,
   uni,
+  uniswap,
   'yearn-vault': yearnVault,
   moloch,
+  unigraph,
```

Comment:
Is it possible to remove the `uniswap` single pool strategy from this PR and rename `unigraph` strategy to `uniswap`? I think the multi pool strategy will be what projects will use, i would prefer if it take the uniswap name.